src/io/message_histogram_collector.hpp

```diff
@@ -39,6 +39,8 @@ struct LatencyHistogramSummary {
   Duration p100;
   Duration sum;
 
+  friend bool operator==(const LatencyHistogramSummary &lhs, const LatencyHistogramSummary &rhs) = default;
```

Comment:
You need to include the `<compare>` header to reliably make this work.

Reply:
applied